crates/pypi_modifiers/src/pypi_tags.rs

```diff
@@ -159,10 +162,37 @@ fn get_implementation_name(python_record: &PackageRecord) -> miette::Result<&'st
     }
 }
 
+/// Return whether the specified record has gil disabled (by being a free-threaded python interpreter)
+fn gil_disabled(python_record: &PackageRecord) -> miette::Result<bool> {
+    // In order to detect if the python interpreter is free-threaded, we look at the depends
+    // field of the record. If the record has a dependency on `python_abi`, then
+    // look at the build string to detect cpXXXt (free-threaded python interpreter).
+    let regex =
```

Comment:
Wont it be enough to just check if it starts with `cp` and ends with `t`?

If not, please use a `OnceLock` to lazily initialize the regex.

Reply:
It can be part of something like this: `*_cp313t` so I belive that the regex is more approriate solution here